examples/remittances.js

```diff
@@ -0,0 +1,51 @@
+const momo = require("../lib");
+
+const { Remittances } = momo.create({ callbackHost: process.env.CALLBACK_HOST });
+
+// initialise collections
+const remittances = Remittances({
+  userSecret: process.env.REMITTANCES_USER_SECRET,
+  userId: process.env.REMITTANCES_USER_ID,
+  primaryKey: process.env.REMITTANCES_PRIMARY_KEY
+});
+
+const partyId = "256776564739";
+const partyIdType = momo.PayerType.MSISDN;
+// Transfer
+remittances
+    .isPayerActive(partyId, partyIdType)
+    .then((isActive) => {
+        console.log("Is Active ? ", isActive);
+        if (!isActive) {
+            return Promise.reject("Party not active");
+        }
+        return remittances.remit({
+            amount: "100",
+            currency: "EUR",
+            externalId: "947354",
+            payee: {
+                partyIdType,
+                partyId
+            },
+            payerMessage: "testing",
+            payeeNote: "hello",
+            callbackUrl: process.env.CALLBACK_URL ? process.env.CALLBACK_URL : "https://75f59b50.ngrok.io"
```

Comment:
maybe leave the ngrok urls out of the code base, just `process.env.CALLBACK_URL` should be okay